synapse/rest/client/versions.py

```diff
@@ -44,7 +44,13 @@ def on_GET(self, request):
                     "r0.5.0",
                 ],
                 # as per MSC1497:
-                "unstable_features": {"m.lazy_load_members": True},
+                "unstable_features": {
+                    "m.lazy_load_members": True,
+                    # Advertise to clients that they need not include an `id_server`
+                    # parameter during registration of password reset, as Synapse now
+                    # decides itself which identity server to use (or none at all).
+                    "m.require_identity_server": True
```

Comment:
also is this intentionally `True`? ie. this is saying that an id server *is* required?

Reply:
Oh, right.

Reply:
This PR is now based on https://github.com/matrix-org/synapse/pull/5835, which has the `account_threepid_delegate` config option.